DMTN-049.tex

```diff
@@ -229,85 +230,131 @@ \section{Technical Considerations from the DM System}\label{sec:dmcon}
+As described in the LSST PZ Roadmap (\url{ls.st/dmtn-049}), DM will select one or more existing, community-vetted algorithms to generate Object PZ which, at least initially, meet a set of minimum scientific attributes and serve the widest variety of science applications.
 
-The goal is for DM to create a shortlist of several community-vetted photo-$z$ estimator(s) that will serve as broad a range of LSST science goals as possible, and also meet the technical considerations (\S~\ref{sec:dmcon}) and the evaluation criteria (\S~\ref{sec:eval}).
-This shortlist would then serve to coalesce the joint efforts of DM and the community during the  "Photo-$z$ Validation Cooperative", which will be based on commissioning data (\S~\ref{sec:pzcoop}). 
+\textbf{LOR Purpose:}
+The Rubin science community has a considerable wealth of expertise in generating PZ catalogs and will be the primary users of the Object PZ data products.
+These letters provide a formal opportunity for the science community to define their minimum scientific needs regarding the LSST Object PZ data product. 
 
-Based on the above goals, these "Letters of Recommendation":
-\begin{itemize}
-\item should address the technical considerations in \S~\ref{sec:dmcon}
-\item should address the evaluation criteria in  \S~\ref{sec:eval}
-\item should be short (i.e., 1-3 pages, ideally), and should not constitute an onerous task
-\item could focus on many, a few, or just one estimator(s) or science goal(s)
-\item could be primarily {\it qualitative} in their evaluation of the photo-$z$ estimators
-\item could include or reference {\it quantitative} analyses of simulated LSST data, or data from other facilities
-\item will be openly shared in the Rubin community
-\end{itemize}
+\textbf{LOR Writers:}
+Any group or individual who would use the LSST Object PZ for their future scientific analyses, and/or are developers of potentially suitable PZ estimators, are encouraged to submit an letter.
+The DM team is especially interested in hearing the needs of scientists who plan to use the LSST Object catalog but who would/could not generate custom PZ estimates.
 
-It is understood that "Letters of Recommendation" from members of the science community who are future users of the {\tt Object} catalog photo-$z$, but who are not developers of photo-$z$ estimators, would focus on recommending third-party software based on their science needs, and might not be able to fully address the technical considerations.
+\textbf{LOR Scope:}
+Letters should recommend minimum attributes of the LSST PZ data product that would enable basic LSST science, describe the science that the LSST PZ data product should enable, and/or qualitatively recommend a certain PZ estimator (or type of estimator).
+Letters do not have to recommend a certain PZ estimator, and can just focus on describing PZ-related science needs.
```

Comment:
Absolutely, I see the merit in this change. I wrote a bit more under scope because managing expectations and preventing people from going way off into huge analyses is important, yet also we don't want people to worry about writing the "wrong kind of LOR":

"Letters could qualitatively (or quantitatively) recommend one or more specific PZ estimators (or type of estimator), but they do not have to: letters could instead recommend minimum attributes of the LSST PZ data product that would enable basic LSST science, or focus on describing the science that the LSST PZ data product should enable.
Discussions on the research and development of new or improved algorithms can be considered as beyond the scope of these letters, as the shortlist will only include currently existing PZ estimators. 
New in-depth quantitative analyses of PZ estimator performance can also be considered as beyond the scope of the LORs because such analyses are the focus of the next stage of the PZ roadmap (see below).
That said, no letters will be rejected for extending beyond these scope boundaries, which are provided just to guide and inform the community's efforts. "

Then I re-wrote the Purpose to better center the recommendations of PZ estimators, too:
"These letters provide a formal opportunity for the science community to advocate for one or more photo-$z$ estimators that will meet their minimum scientific needs, and/or to define these minimum needs so that DM can consider them when shortlisting PZ estimators."